CI build fails as change detected if a new test is added.

```
Failed
=== RUN   TestRelationalDeletionOfADocumentUsingSingleKey_Success
2022-04-07T22:14:29.315Z, �[33mWARN�[0m, defra.db.tests, tests/utils.go:278, Relational delete mutation where one element exists., {"Database": "badger-file-system"}
2022-04-07T22:14:29.315Z, �[34mINFO�[0m, defra.db, db/db.go:203, Closing DefraDB process...
2022-04-07T22:14:29.320Z, �[34mINFO�[0m, defra.db, db/db.go:208, Successfully closed running process
2022-04-07T22:14:29.320Z, �[31mERROR�[0m, defra.db.tests, tests/utils.go:558, , {"Error": "chdir /tmp/defra/6f56272db758eb6b16addcdac1ae61f45d2430f1/code/db/tests/mutation/relation/delete: no such file or directory"}
--- FAIL: TestRelationalDeletionOfADocumentUsingSingleKey_Success (0.01s)
panic: chdir /tmp/defra/6f56272db758eb6b16addcdac1ae61f45d2430f1/code/db/tests/mutation/relation/delete: no such file or directory [recovered]
	panic: chdir /tmp/defra/6f56272db758eb6b16addcdac1ae61f45d2430f1/code/db/tests/mutation/relation/delete: no such file or directory

goroutine 82 [running]:
testing.tRunner.func1.2({0xbb0300, 0xc00021b7d0})
	/usr/local/go/src/testing/testing.go:1209 +0x24e
testing.tRunner.func1()
	/usr/local/go/src/testing/testing.go:1212 +0x218
panic({0xbb0300, 0xc00021b7d0})
	/usr/local/go/src/runtime/panic.go:1038 +0x215
github.com/sourcenetwork/defradb/db/tests.setupDatabaseUsingTargetBranch({0xd8ae10, 0xc00003c058}, 0xc0002aa680, {{0xc7ae35, 0x12}, {0xc0000f0910, 0x4a}, {0xd9e860, 0xc00016fe30}, {0xd97fd8, ...}}, ...)
	/home/circleci/repo/db/tests/utils.go:559 +0x645
github.com/sourcenetwork/defradb/db/tests.ExecuteQueryTestCase(0xf26db7, {0xc9fb94, 0xf9}, {0xc0001f7cd8, 0x3, 0x3}, {{0xc966ed, 0x34}, {0xc9e59c, 0x6b}, ...})
	/home/circleci/repo/db/tests/utils.go:286 +0x37a
github.com/sourcenetwork/defradb/db/tests/mutation/relation.ExecuteTestCase(...)
	/home/circleci/repo/db/tests/mutation/relation/utils.go:42
github.com/sourcenetwork/defradb/db/tests/mutation/relation/delete.TestRelationalDeletionOfADocumentUsingSingleKey_Success(0x0)
	/home/circleci/repo/db/tests/mutation/relation/delete/single_id_test.go:167 +0x90e
testing.tRunner(0xc0002aa680, 0xca9b00)
	/usr/local/go/src/testing/testing.go:1259 +0x102
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1306 +0x35a
```

--- a/db/tests/utils.go
+++ b/db/tests/utils.go
@@ -552,11 +552,20 @@ func setupDatabaseUsingTargetBranch(
 	out, err := goTestCmd.Output()
 
 	if err != nil {
-		// Only log the output if there is an error, logging child test runs confuses
-		// the go test runner making it think there are no tests in the parent
-		// run (it will still run everything though)!
-		log.ErrorE(ctx, string(out), err)
-		panic(err)
+		// If file is not found - this must be a new test and
+		// doesn't exist in the target branch, so we pass it
+		// because the child process tries to run the test, but
+		// if it doesnt find it, the parent test should pass (not panic).
+		if strings.Contains(err.Error(), ": no such file or directory") {
+			t.SkipNow()
+		} else {
+			// Only log the output if there is an error different from above,
+			// logging child test runs confuses the go test runner making it
+			// think there are no tests in the parent run (it will still
+			// run everything though)!
+			log.ErrorE(ctx, string(out), err)
+			panic(err)
+		}
 	}
 
 	refreshedDb, err := newBadgerFileDB(ctx, t, dbi.path)

